test/test_utils/ec2.py

```diff
@@ -452,6 +458,44 @@ def get_ec2_instance_tags(instance_id, region=DEFAULT_REGION, ec2_client=None):
     return {tag["Key"]: tag["Value"] for tag in response.get("Tags")}
 
 
+def execute_asynchronus_testing_using_s3_bucket(
+    connection,
+    execution_command,
+    connection_timeout,
+    s3_location,
```

Comment:
I'd suggest to remove a bucket name from here. Otherwise we'll end up with a bunch of buckets those we don't really control. Create 1 bucket in each environment. E.g. dlc-test-asynch-execution-<acc_number>. And 
a. let people to create directories inside of that bucket;
b. implement a logic similar to [this](https://github.com/aws/deep-learning-containers/blob/6d2b86727e79b2cb4e5ba66456f8d5369b952934/test/test_utils/pytest_cache.py#L153)

I personally would prefer (b). It will give us more control. 

BTW! Don't forget to setup a retention policy for that bucket. "Delete after 1 week / 1 month" or something like that.

Reply:
I have created a bucket that uses the account number so that we do not have to feel the pain of running it in different accounts.